MAINTAINERS.md

```diff
@@ -0,0 +1,84 @@
+This file explains the build and packaging commands of azure-chef-extension.
+
+##Contributors

```

Comment:
This should be a maintainers section, and it should include me :smile:. Also, you may not want to add your entire team as maintainers if everyone hasn't spent a lot of time on it.